dali/daliadmin/daadmin.hpp

```diff
@@ -28,8 +28,9 @@ extern DALIADMIN_API void xmlSize(const char *filename, double pc);
 extern DALIADMIN_API void translateToXpath(const char *logicalfile, DfsXmlBranchKind tailType = DXB_File);
 
 extern DALIADMIN_API void _export_(const char *path, const char *dst, bool safe = false);
-extern DALIADMIN_API void import(const char *path, const char *src, bool add);
-extern DALIADMIN_API void _delete_(const char *path, bool backup);
+extern DALIADMIN_API bool import(const char *path, const char *src, bool add, StringBuffer &out);
+extern DALIADMIN_API bool importXML(const char *path, const char *xml, bool add, StringBuffer &out);
```

Comment:
the method names could do with improving to make it obvious what they do, how about -
importFromFile (and change 'src' to 'filename' in header and definition)  and,
importFromXML

Reply:
Will change